auth.js

```diff
@@ -0,0 +1,69 @@
+/* login, signup 공통 */
+
+// 이메일 유효성 검사 함수
+function emailValidation(email) {
+    const emailRegex = /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/;
+    return emailRegex.test(email);
+}
+
+// 요소 가져오기
+const emailInput = document.getElementById("email");
+const passwordInput = document.getElementById("password");
+const emailErr = document.getElementById("emailErr");
+const passwordErr = document.getElementById("passwordErr");
+const passwordVisibility = document.getElementById("passwordVisibility");
```

Comment:
💊 제안
passwordInput처럼 변수명을 통해 해당 변수가 어떤 타입의 값을 알 수 있도록 명명하시는 것이 가독성 측면에서 좋습니다~

```suggestion
const passwordVisibilityButton = document.getElementById("passwordVisibility");
```